issues.md

```diff
@@ -1,156 +1,150 @@
-
-As a rule, do:
-
-- Use [GitHub discussion](https://github.com/mdn/mdn-community/discussions) before filing an issue. This helps to keep issues focused and productive.
-- Ask questions using other mechanisms like [chat rooms](https://chat.mozilla.org/#/room/#mdn:mozilla.org) or [forums](https://discourse.mozilla.org/c/mdn/236) if you are not sure whether something is useful or have a simple question.
-- Read our [contribution guidelines](/en-US/docs/MDN/Community) and [writing guidelines](/en-US/docs/MDN/Writing_guidelines) first to try to solve the issue yourself.
-
-Don't:
+  - **Issue title** must convey succinctly the _required action_.
 
-- Complicate issues by trying to discuss multiple topics at once, or making off-topic comments.
-- Open lots of issues asking vague questions.
-- Ask questions without trying to solve the problem yourself first.
+  - **Issue description** must clearly describe the bug and the action required to resolve the issue. It must also list the task or sub-tasks to be completed to resolve the issue. Some other guidelines include:
+    - Use the description field to indicate the status of the task or sub-tasks by using checklists.
+    - Updates to the status of a task should be conveyed by editing the task list in the issue description instead of adding a new comment. One should not have to scroll through comments or replies to find the status of a task or sub-task.
+    - Comments in an issue should be limited to details or context that help resolve the issue.
 
-## Working on an issue
+- If the information you provide in the issue is incomplete, then you might be contacted later during the [issue triaging process](#review_issue_to_determine_completeness_of_information).
 
-All repositories have an issue tracker, where you can find tasks to help contribute
+- If you find yourself in one of the following situations, move the conversation to [MDN's discussion on GitHub](https://github.com/mdn/mdn-community/discussions):
+  - A discussion needs to take place to clarify an issue.
+  - A discussion begins after opening the issue.
+  - The issue has no clear consensus on its resolution.
+  - The requirements for completing the task expand while it's being resolved or the work is unclear.
 
-Most repositories have a `help-wanted` label or `good-first-issue` label. Some do not, but this is not a pre-requisite and you are welcome to browse and pick something that is suitable for your skill set.
+- You can open an issue and [fix it yourself](#fixing_issues_yourself).
 
-Once you've found an issue you'd like to work on, make sure no one else is assigned to the issue. Add a comment saying you would like to work on it and assign the issue to yourself.
+### Creating an issue task tracker
```

Comment:
I think we should use "task list" terminology instead of "tracker" so that we reflect GitHub terminology and docs https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists